src/Sylius/Behat/Context/Api/Admin/ManagingProductAssociationTypesContext.php

```diff
@@ -285,4 +285,32 @@ public function iRemoveItsNameFromTranslation(string $localeCode): void
             ],
         ]);
     }
+
+    /**
+     * @When I sort the product associations :sortType by code
+     */
+    public function iSortProductAssociationsByCode(string $sortType = 'ascending'): void
```

Comment:
In one context you use `sortingOrder` and in the other `sortType`, it would be good to use one keyword for that 🖖🏻 
```suggestion
    public function iSortProductAssociationsByCode(string $sortingOrder = 'ascending'): void
```
🤔